docs/lit/examples/2_Plots.jl

```diff
@@ -18,44 +18,83 @@ using Plots, Unitful, UnitfulRecipes
 
 # ## Lines
 
-plot(Plots.fakedata(50, 5) * u"m", w=3)
+plot(Plots.fakedata(50, 5) * u"m", w = 3)
 
 # ## Parametric plots
 
-plot(t -> sin(t)*u"s", t -> sin(2t)*u"m", 0, 2π, line=4, leg=false, fill=(0, :orange))
+plot(
+    t -> sin(t) * u"s",
+    t -> sin(2t) * u"m",
+    0,
+    2π,
+    line = 4,
+    leg = false,
+    fill = (0, :orange),
+)
 
 # ## Colors
 
-y = rand(100)*u"km"
-plot((0:10:100)*u"hr", rand(11, 4)*u"km", lab="lines", w=3, palette=:grays, fill=0, α=0.6)
-scatter!(y, zcolor=abs.(y .- 0.5u"km"), m=(:heat, 0.8, Plots.stroke(1, :green)), ms=10 * abs.(y .- 0.5u"km") .+ 4u"km", lab="grad")
+y = rand(100) * u"km"
+plot(
```

Comment:
Same as above, these examples were nice and compact and look a bit worse to me now.